index.js

```diff
@@ -77,7 +79,7 @@ const generateQrCode = data => {
 		data.bic,
 		data.name,
 		serializeIBAN(data.iban),
-		'EUR' + data.amount.toFixed(2),
+		data.amount ? 'EUR' + data.amount.toFixed(2) : data.amount,
```

Comment:
This will not put `EUR` if `data.amount` is `0`.

Reply:
Hi @derhuerst ,

If amount is 0 an error is thrown from a few lines above 'data.amount must be >=0.01 and <=999999999.99.'.

Best,
Tamas